src/resources/i18n/translations/en.json

```diff
@@ -96,7 +96,7 @@
 	"ARIA HINT - edit this": "Edit this",
 	"ARIA HINT - episode duration": "Episode duration",
 	"ARIA HINT - go back to the login screen": "Go back to the login screen",
-	"ARIA HINT - go to the Bitcoin wallet setup screen": "Go to the setup for ₿ wallets",
+	"ARIA HINT - go to the Bitcoin wallet setup screen": "Go to the setup for Bitcoin wallets",
```

Comment:
Part of using the symbol is getting away from "bitcoin/Bitcoin", where I think "bitcoin" is the correct one to use, but alas.
Not a big issue.

Reply:
👍